README.md

```diff
@@ -43,23 +43,29 @@ npm install
 npm run build:prod:watch
```

Comment:
Now that I re-read the readme I believe we should switch out this instruction with the following since it will run everything in development mode which provides better stack traces and logging.

```suggestion
ENV=production npm run build:watch ´
```